src/amber/controller/helpers/responders.cr

```diff
@@ -35,13 +45,14 @@ module Amber::Controller::Helpers
       end
 
       def body
-        @available_responses[type]
+        @available_responses[type]?
       end
 
       private def select_type
         @type ||= begin
           raise "You must define at least one response_type." if @available_responses.empty?
-          @requested_responses << @available_responses.keys.first
+          # NOTE: If only one response is requested don't return something else.
+          @requested_responses << @available_responses.keys.first if @requested_responses.size != 1
```

Comment:
So, this is "non-strict" mode - where even if they didn't explicitly request the type we return the first available response (if they had more than one thing in the 'accept' header), correct?

Reply:
While I feel that if json is the only thing available it should be return even if the browser is asking for html, it seems like a different case if someone asks for `domain.com/cats.json`. In that case it will return 406 if json doesn't exist.